articles/tokens/access-token.md

```diff
@@ -0,0 +1,142 @@
+* [Examples using auth0.js](https://github.com/auth0/auth0.js).
+* Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
+
+## How to use an access token
+
+Access tokens are typically obtained in order to access user-owned resources. For example, a Calendar application on a mobile phone needs access to a Calendar API in the cloud in order to read the user's scheduled events and create new events.
+
+Such access is requested by the application and granted by the user. For example, when using Lock, access to read and write the user's calendar can be requested by calling `show()`:
+
+```js
+lock.show({
+  responseType: 'token',
+  authParams: {
+    scope: 'read write'
+  }
+});
+```
+
+In this case the user will be prompted to permit read and write access. If allowed, an access token will be issued to the client, which the client can then use when making requests to the Calendar API. If consent has already been granted by the user, no consent dialog will be displayed and the access token will be issued without additional prompts.
+
+Also, the consent dialog might be displayed again if the access level changes. For example, if the user has granted read access but the functionality changes so write access is required as well, the user will have to use the consent dialog to grant the additional access.
+
+In some cases, consent can also be pre-configured administratively. This typically occurs when the user is an employee, and there are a set of company-specific applications that are always allowed access to employee data.
+
+### Server-to-server interactions
+
+Access tokens can also be issued directly to applications, permitting access to APIs that don't provide user-specific data. Such scenarios involve server-to-server interactions.
+
+For example, a reverse geocoding API that accepts a latitude, longitude coordinate and returns a readable place name does not access user-owned data. In such cases a backend server needs to call the geocoding API in order to perform the translation.
+
+Server-to-server access tokens can be obtained using the `client_credentials` flow, exchanging the client's credentials for an API access token.
+
+```
+POST /oauth/token
+Host​: {tenant}.auth0.com
+Content-Type: application/x-www-form-urlencoded
+
+grant_type=client_credentials&
+client_id={CLIENT_ID}&
+client_secret={CLIENT_SECRET}
+audience=https://api.example.com/geocoding/v1/
+```
+
+The result will be an access token that can be used to make requests to the geocoding API.
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "access_token":"ey...",
+  "token_type":"Bearer",
+  "expires_in":86400,
+  "example_parameter":"example_value"
+}
+```
+
+In order to obtain this access token, the client must first have been granted permission to access the geocoding API. This is typically done by requesting access from the administrator of the geocoding API.
```

Comment:
I'll add a link [to the doc](https://auth0.com/docs/api-auth/config/using-the-auth0-dashboard) that offers more details on this.